ice40/boards.cmake

```diff
@@ -5,14 +5,21 @@ define_board(
   DEVICE lp1k
   PACKAGE qn84
   PROG_TOOL ${ICEPROG_TOOL}
-  PROG_CMD ${ICEPROG_TOOL} -ew
+  PROG_CMD "${ICEPROG_TOOL} -ew"
 )
 
 define_board(
   BOARD hx8k-b-evn
   DEVICE hx8k
   PACKAGE ct256
   PROG_TOOL ${ICEPROG_TOOL}
-  PROG_CMD ${ICEPROG_TOOL} -S
+  PROG_CMD "${ICEPROG_TOOL} -S"
+)
+
+define_board(
+  BOARD icevision
+  DEVICE up5k
+  PACKAGE sg48
+  PROG_TOOL ${ICEPROG_TOOL}
 )
 
```

Comment:
I think your still missing a couple of boards from https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/ice40/make/boards.mk?

Can you port across the comments too?

Reply:
I think we should probably also have an empty / dummy board?

Reply:
Added dummy board.